Makefile

```diff
@@ -55,16 +66,8 @@ package-%:
 
 	@mv -v ../delphix-platform-$*_*_amd64.deb artifacts
 
-SHELL_SCRIPTS := \
-	debian/postinst \
-	debian/prerm \
-	usr/bin/download-latest-image \
-	usr/bin/unpack-image \
-	var/lib/delphix-platform/ansible/apply \
-	var/lib/delphix-platform/os-migration
-
 shellcheck:
-	shellcheck $(SHELL_SCRIPTS)
+	shellcheck $$(shfmt -f .)
 
 shfmtcheck:
-	! shfmt -d $(SHELL_SCRIPTS) | grep .
+	shfmt -d .
```

Comment:
Did you verify this will fail when it detects a problem? i.e. the TravisCI checks will properly report the failure?

I'd much rather to see this split into a separate commit, with a test commit to verify the failure modes still work correctly.

Reply:
I've just tested it and it works with travis